third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc

```diff
@@ -293,6 +264,8 @@ function isValidBaseURL(url) {
 // message.js
 ///////////////////////////////////////////////////////////////////////////////
 
+const CDPError_MissingContext = 1001;
```

Comment:
Can we avoid copy-and-pasting here? Maybe use `DefineProperty` to add a `Constants` object to `__RECORD_REPLAY_ARGUMENTS__`?

Reply:
Nice!  Didn't notice that; done.